client/src/main/webjar/app.js

```diff
@@ -8,49 +8,158 @@
         ]
     );
 
-    function User(
+    function ExampleController(
```

Comment:
I have a skeleton client code about ready to go in, but I'll let you merge this first, and I will just absorb capturing the $resources from this into my branch.